pydantic_core/_pydantic_core.pyi

```diff
@@ -174,3 +174,17 @@ def list_all_errors() -> 'list[ErrorTypeInfo]':
     """
     Get information about all built-in errors.
     """
+
+class ValidatorInfo:
+    """
+    Argument passed to validation functions.
+    """
+
+    field: str
```

Comment:
I don't see this on the rust struct, am I missing something?

Reply:
Good catch! I was planning on adding it and forgot. @samuelcolvin I don't see any easy way to access the field name for a validator. Is it okay if I add that to `FunctionBuilder` and the `CoreSchema`s? Do we want to pass in just the field name or some other object (the schema?).